searx/settings.yml

```diff
@@ -478,7 +478,9 @@ engines:
 
   - name : invidious
     engine : invidious
-    base_url : 'https://invidio.us/'
+    base_url : 
```

Comment:
There is an extra space after `:`. Please remove it.

Reply:
Hello,
I did it.
Thx.